nextmv-gurobipy/nextmv_gurobipy/options.py

```diff
@@ -6,6 +6,21 @@
 
 import nextmv
 
+SKIP_PARAMETERS = [
```

Comment:
Maybe I am missing something, but why are we skipping a number of these? The `InputFile` and maybe the `Tune*` ones I understand (mostly), but why the others? :thinking:

Reply:
The CLI only ones can only be used with the Gurobi CLI directly, so even though the convenience function returns the parameter, we can’t use it from this SDK. For the `Cutoff`, the issue is that it requires a value of `+Inf` or `-Inf`, which:
1. We know is not reliable across computers.
2. Should be set according to the sense of the model. At the point of Option creation, we have not created the model yet, so we can’t tell.

IN any case, the option can always be set after the fact.